package main.java.edugit.exceptions;

/**
 * An error thrown when a push doesn't go through because
 * Remote is ahead of Local.
 */
public class PushToAheadRemoteError extends Exception {
}
